spec/unit/generator/xcconfig/pod_xcconfig_spec.rb

```diff
@@ -159,7 +159,7 @@ module XCConfig
             @coconut_pod_target.test_dependent_targets = [@banana_pod_target]
             generator = PodXCConfig.new(@coconut_pod_target, true)
             xcconfig = generator.generate
-            xcconfig.to_hash['LIBRARY_SEARCH_PATHS'].should == '$(inherited) "$PODS_CONFIGURATION_BUILD_DIR/BananaLib" $(inherited) "${PODS_ROOT}/../../spec/fixtures/banana-lib"'
+            xcconfig.to_hash['LIBRARY_SEARCH_PATHS'].should == '$(inherited) "$PODS_CONFIGURATION_BUILD_DIR/BananaLib" "$PODS_CONFIGURATION_BUILD_DIR/CoconutLib" $(inherited) "${PODS_ROOT}/../../spec/fixtures/banana-lib"'
```

Comment:
Also another PR will be to include `recursive_test_dependent_targets` instead of just the direct dependencies.